keep-runtime/src/main.rs

```diff
@@ -18,6 +18,19 @@
 //!         ),
 //!     ]
 //! ```
+//! ## Example build and invocation with binary
+//!
+//! ```console
+//! $ cargo build --release
+//! Finished release [optimized] target(s) in 0.06s
+//! $ RUST_LOG=keep_runtime=info RUST_BACKTRACE=1 ./target/x86_64-unknown-linux-musl/release/
+//!    keep-runtime fixtures/wasi_snapshot1.wat
```

Comment:
s/wat/wasm/ ? Note that the `.wasm` file is now generated under `OUT_DIR` at build time.